regex-assembly/include/932130.ra

```diff
@@ -1,16 +1,16 @@
 ##! Please refer to the documentation at
 ##! https://coreruleset.org/docs/development/regex_assembly/.
 
-\$\(.*\)
-\$\{.*\}
-<\(.*\)
->\(.*\)
+\$\([^()]*\)
+\$\{[^{}]*\}
+<\([^()]*\)
+>\([^()]*\)
```

Comment:
```suggestion
\$\([^\(\)]*\)
\$\{[^\{\}]*\}
<\([^\(\)]*\)
>\([^\(\)]*\)
```

Reply:
Escaping meta characters inside of character classes is unnecessary, except for `]` (and for us `"` additionally, because we use the regular expressions in a double quoted string).